### PR TITLE
Updated Key Vault CHANGELOG and README for February Beta release

### PR DESCRIPTION
**Pull request by vcolin7**

### CODE REVIEW

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Maven dependency for the Azure Key Vault Certificate client library. Add it to y
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-certificates</artifactId>
-    <version>4.1.4</version>
+    <version>4.2.0-beta.3</version>
```
Comment (g2vinay) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:17]: best to let READMEs point at stable release.

Comment (g2vinay) [reply sdk/keyvault/azure-security-keyvault-certificates/README.md:17]: below too

**Review by g2vinay — APPROVED** [review]
READMEs should point to the stable release. So, that customers try the latest stable release with confidence and not get nervous to see the beta as the recommended package.

File: sdk/keyvault/azure-security-keyvault-certificates/README.md (hunk 1 above)
Comment (vcolin7) [reply sdk/keyvault/azure-security-keyvault-certificates/README.md:17]: Should we do that even when releasing beta?

**Pull request merged**